ksql-functional-tests/src/test/resources/query-validation-tests/array-functions.json

```diff
@@ -0,0 +1,110 @@
+{
+  "comments": [
+    "Tests covering the use of the ROUND(DOUBLE), FLOOR(DOUBLE), CEIL(DOUBLE) functions."
```

Comment:
Given this tests maths functions - why is the test file `array-functions`?

There is already a `math.json`. How about adding test cases to that file?

Reply:
Ok, so first someone suggested I rename it to delimited-value-format.json, so I did. Then you suggested I rename it to delimited.json so I did. Now I should move everything to math.json. Not a problem, but hopefully this is the last move! ;)

Reply:
It's actually in udf-test.json now. Not all the udfs will be related to maths.